pkg/kube/snapshot/snapshot_test.go

```diff
@@ -1458,7 +1607,83 @@ func (s *SnapshotLocalTestSuite) TestLabels(c *C) {
 		}
 	}
 }
-
+func (s *SnapshotLocalTestSuite) TestAnnotations(c *C) {
+	ctx := context.Background()
+	scheme := runtime.NewScheme()
+	scheme.AddKnownTypeWithName(schema.GroupVersionKind{Group: "snapshot.storage.k8s.io", Version: "v1alpha1", Kind: "VolumeSnapshotList"}, &unstructured.UnstructuredList{})
+	scheme.AddKnownTypeWithName(schema.GroupVersionKind{Group: "snapshot.storage.k8s.io", Version: "v1beta1", Kind: "VolumeSnapshotList"}, &unstructured.UnstructuredList{})
+	scheme.AddKnownTypeWithName(schema.GroupVersionKind{Group: "snapshot.storage.k8s.io", Version: "v1", Kind: "VolumeSnapshotList"}, &unstructured.UnstructuredList{})
+	ns := "namespace"
+	volName := "vol1"
+	snapName := "snap1"
+	snapClass := "snapClass"
+	fakeCli := fake.NewSimpleClientset(fakePVC(volName, ns))
+	for _, tc := range []struct {
+		dynCli              dynamic.Interface
+		snapshotAnnotations map[string]string
+		expectedAnnotations map[string]string
+		errChecker          Checker
+	}{
+		{
+			dynCli: dynfake.NewSimpleDynamicClient(scheme),
+			snapshotAnnotations: map[string]string{
+				"annotationtest": "true",
+			},
+			expectedAnnotations: map[string]string{
+				"annotationtest": "true",
+			},
```

Comment:
[nit] We might not need `expectedAnnotations` as they are always same as `snapshotAnnotations`. We can use `snapshotAnnotations` in assert checker

Reply:
done